ios/xcode/MGLKit/MGLLayer.mm

```diff
@@ -710,6 +711,18 @@ - (void)ensureSurfaceCreated
             red = green = blue = alpha = 8;
             colorSpace                 = EGL_GL_COLORSPACE_SRGB_KHR;
             break;
+        case MGLDrawableColorFormatRGBA16:
+            red = green = blue = alpha = 16;
+            switch (_drawableColorSpace)
+            {
+                case MGLDrawableColorSpaceBT2020PQ:
+                    colorSpace = EGL_GL_COLORSPACE_BT2020_PQ_EXT;
```

Comment:
This was failing with `Failed to call eglCreateWindowSurface()` because of missing validation for these new formats.